sections/builder.js

```diff
@@ -1526,28 +1526,42 @@ export default [
   {
     type: "method",
     method: "offset",
-    example: ".offset(value)",
-    description: "Adds an offset clause to the query.",
+    example: ".offset(value, options={skipBinding: boolean})",
+    description: "Adds an offset clause to the query. An optional skipBinding parameter may be specified to doesn't add offset as a prepared value (some database doesn't allow prepared value for offset).",
     children: [
       {
         type: "runnable",
         content: `
           knex.select('*').from('users').offset(10)
         `
+      },
+      {
+        type: "runnable",
+        content: `
+          // Offset value isn't a prepared value.
+          knex.select('*').from('users').offset(10, {skipBinding: true}).toSQL().sql
+        `
       }
     ]
   },
   {
     type: "method",
     method: "limit",
-    example: ".limit(value)",
-    description: "Adds a limit clause to the query.",
+    example: ".limit(value, options={skipBinding: boolean})",
+    description: "Adds a limit clause to the query. An optional skipBinding parameter may be specified to doesn't add limit as a prepared value (some database doesn't allow prepared value for limit).",
```

Comment:
```suggestion
    description: "Adds a limit clause to the query. An optional skipBinding parameter may be specified to avoid adding limit as a prepared value (some databases don't allow prepared values for limit).",
```